WordPress/WordPressTest/TestUtilities/JSONLoader.swift

```diff
@@ -4,13 +4,12 @@ import Foundation
 @objc open class JSONLoader: NSObject {
     public typealias JSONDictionary = Dictionary<String, AnyObject>
 
-    /**
-    *  @brief      Loads the specified json file name and returns a dictionary representing it.
-    *
-    *  @param      path    The path of the json file to load.
-    *
-    *  @returns    A dictionary representing the contents of the json file.
-    */
+    /// Loads the specified json file and returns a dictionary representing it.
+    ///
+    /// - Parameters:
+    ///   - name: The name of the file
+    ///   - type: The extension of the file
+    /// - Returns: A dictionary representing the contents of the json file.
```

Comment:
One small nitpick: In other parts of the PR, "json" appear all uppercase. As far as I understand, JSON is the appropriate way to write the word because it's an acronym.

```suggestion
    /// Loads the specified JSON file and returns a dictionary representing it.
    ///
    /// - Parameters:
    ///   - name: The name of the file
    ///   - type: The extension of the file
    /// - Returns: A dictionary representing the contents of the JSON file.
```